### PR TITLE
8 Create event

### LINKED ISSUE #8: Create event
At backend the form values will be passed and validated and stored in DB.

### PR DESCRIPTION
**Pull request by akosgdcsi**
-Data entities and connections changed accordingly ER_diagram

-refactoring for logout had to be done in order to identityuser's id to be int

-automapper added for converting DTO's to data entities 

-repositories for each entities created for backend work to be done simultaneously 

-create event logic and controller created for the issue

### CODE REVIEW

**Review by rolandbogdan — CHANGES_REQUESTED** [review]
PLEASE ÁKOS STOP REQUESTING MY REVIEWS ON YOUR PULL REQUESTS

**Comment by tubamark98** [comment]
Bence do your part already.

File: wman-backend/Wman.Data/wmanDb.cs [hunk 1]
```diff
@@ -47,8 +47,9 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
               .HasForeignKey(ur => ur.RoleId)
               .IsRequired();
 
-            modelBuilder.Entity<IdentityRole>().HasData(
-                new { Id = "0d301757-99d2-4253-aac2-39e298dd0ab7", Name = "Debug", NormalizedName = "DEBUG" }
+
+
+
```
Comment (rolandbogdan) [inline wman-backend/Wman.Data/wmanDb.cs:75]: Remove several consecutive empty lines

File: wman-backend/Wman.Logic/Helpers/AutoMapperProfiles.cs [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+{
+    public class AutoMapperProfiles : Profile
+    {
+        public AutoMapperProfiles()
+        {
+            CreateMap<IdentityResult, UserDTO>();
+            CreateMap<CreateEventDTO, AddressHUNDTO>();
+            CreateMap<AddressHUNDTO, AddressHUN>();
+            CreateMap<CreateEventDTO, WorkEvent>()
+    // chose the destination-property and map the source itself
```
Comment (rolandbogdan) [inline wman-backend/Wman.Logic/Helpers/AutoMapperProfiles.cs:21]: Fix identation of comment. Start comment with capital letter, and use correct grammar

**Review by rolandbogdan — CHANGES_REQUESTED** [review]
Make it better

File: wman-backend/Wman.Data/wmanDb.cs (hunk 1 above)
Comment (akosgdcsi) [reply wman-backend/Wman.Data/wmanDb.cs:75]: right on chief 🤙🏻

File: wman-backend/Wman.Logic/Helpers/AutoMapperProfiles.cs (hunk 2 above)
Comment (tubamark98) [reply wman-backend/Wman.Logic/Helpers/AutoMapperProfiles.cs:21]: Very cool and all but i don't remember asking

Comment (akosgdcsi) [reply wman-backend/Wman.Logic/Helpers/AutoMapperProfiles.cs:21]: I dare to say no 🚳 ⛔

Comment (tsaskater) [reply wman-backend/Wman.Logic/Helpers/AutoMapperProfiles.cs:21]: Please go to your own project->[here](https://shorturl.at/pqzEZ)

**Review by luxz97 — APPROVED** [review]

**Review by pop-simon — APPROVED** [review]

**Review by tsaskater — APPROVED** [review]
Good Job!
@akosgdcsi @luxz97 both of you did good.😎😉

**Pull request merged**